README.md

```diff
@@ -143,9 +143,20 @@ This section describes how to get started developing Chisel itself, including ho
 
 ### Compiling and Testing Chisel
 
-In the chisel3 repository directory compile the Chisel library:
+First, clone and build the master branch of [FIRRTL](https://github.com/freechipsproject/firrtl) and [Treadle](https://github.com/freechipsproject/treadle), as the master branch of Chisel may depend on unreleased changes in those projects:
 
 ```
+git clone https://github.com/freechipsproject/firrtl.git
+git clone https://github.com/freechipsproject/treadle.git
+pushd firrtl; sbt +publishLocal; popd
+pushd treadle; sbt +publishLocal; popd
```

Comment:
I think the `+` is unnecessary since the major version of Scala is the same for all. Granted I expect when we finally update to 2.13 there will be a brief period where this would be necessary.

Reply:
Fair call. I will drop it.

Reply:
Done